src/style.css

```diff
@@ -1,3 +1,42 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 300;
+  font-style: normal;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 50px;
+  background: #fff;
+  gap: 419px;
```

Comment:
weird gap, don't use such values, you already have space-between to add intent between header items